src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/Connection.php

```diff
@@ -346,6 +349,13 @@ public function send(string $body, array $headers, int $delay = 0, ?string $mess
             ]);
         }
 
+        if (null !== $xrayTraceId) {
+            $parameters['MessageSystemAttributes'][self::AWS_XRAY_TRACE_HEADER_NAME] = new MessageSystemAttributeValue([
```

Comment:
I'd rather use the official constant (https://github.com/async-aws/aws/blob/c0fa8720bfee1c9dc0568580ebd52bf9f284282b/src/Service/Sqs/src/Enum/MessageSystemAttributeName.php#L10) instead of defining a condt in this bridge.

Reply:
Wow, nice.